osquery/events/events.cpp

```diff
@@ -419,6 +419,7 @@ Status EventFactory::run(EventPublisherID& type_id) {
   // The runloop status is not reflective of the event type's.
   publisher->tearDown();
   VLOG(1) << "Event publisher " << publisher->type() << " runloop terminated";
+  VLOG(1) << "Reason Given: " << status.getMessage();
```

Comment:
It's best to log on one line, then folks monitoring the log outputs can tokenize error/warning/info messages per line. VLOG -> INFO when using --debug.